### PR TITLE
chore: add new section for null and non-null filter conditions

### LINKED ISSUE #4667: ORM query: example for field == null
There is nowhere mentioned, how to ORM query a table for entries having a certain field set (i.e. NOT NULL).

### PR DESCRIPTION
**Pull request by ruheni**
Closes  #4667

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx [hunk 1]
```diff
@@ -123,6 +123,30 @@ const result = await prisma.user.findMany({
...
+const posts = await prisma.post.findMany({
+  where: {
+    content: null,
+  },
+})
+```
+
+### Filter for non-null fields
+
+The following query returns all post returns all posts whose `content` field **not** `null`:
```
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx]:
```suggestion
The following query returns all posts whose `content` field is **not** `null`:
```

File: content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx [hunk 2]
```diff
@@ -123,6 +123,30 @@ const result = await prisma.user.findMany({
 </cmdResult>
 </CodeWithResult>
 
+### Filter on null fields
+
+The following query returns all post returns all posts whose `content` field is `null`:
```
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx]:
```suggestion
The following query returns all posts whose `content` field is `null`:
```

**Review by jharrell — APPROVED** [review]

File: content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx [hunk 3]
```diff
@@ -123,6 +123,30 @@ const result = await prisma.user.findMany({
...
+    content: null,
+  },
+})
+```
+
+### Filter for non-null fields
+
+The following query returns all posts whose `content` field is **not** `null`:
+
+```
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx]:
```suggestion
```ts
```

File: content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx [hunk 4]
```diff
@@ -123,6 +123,30 @@ const result = await prisma.user.findMany({
...
 
+### Filter on null fields
+
+The following query returns all posts whose `content` field is `null`:
+
+```ts
+const posts = await prisma.post.findMany({
+  where: {
+    content: null,
+  },
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx]: comma here, no comma on example below (on any of the spots where one could put them).

Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx]:
```suggestion
  where: {
    content: null
  }
```

**Pull request merged**